purefa/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Can you add the `purefa.openmetrics.health` service check information here? Here's an example: https://github.com/DataDog/integrations-core/blob/master/pulsar/assets/service_checks.json

Reply:
Ack